src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -269,6 +271,19 @@ private function registerLoaders(ContainerBuilder $container, array $bundles)
     }
 
     /**
+     * Manipulate doctrine extension services according to the configuration.
+     *
+     * @param ContainerBuilder $container
+     * @param array            $config
+     */
+    private function registerDoctrineExtensionConfiguration(ContainerBuilder $container, array $config)
```

Comment:
The function name reveals a problem with the configuration structure.

It should probably be something like:

```yaml
api_platform:
    doctrine:
        eager_loading: true
```